.github/workflows/send-issue-notification.yml

```diff
@@ -0,0 +1,27 @@
+name: Send New Actions update Alert on Teams
+
+on:
+  issues:
+    types: [opened,labeled]
+    
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  send-notification:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repo 
+      uses: actions/checkout@v3
```

Comment:
Why are you checking out the repo here? There is no file from the repo used in this workflow, so this is not needed.